### PR TITLE
KOGITO-5744 example with messaging using process incubation api

### PR DESCRIPTION
**Pull request by tiagodolphine**
Many thanks for submitting your Pull Request :heart:! 

Please make sure that your PR meets the following requirements:

**WARNING! Please make sure you are opening your PR against `main` branch!**

- [ ] You have read the [contributors guide](https://github.com/kiegroup/kogito-runtimes#contributing-to-kogito)
- [ ] Pull Request title is properly formatted: `KOGITO-XYZ Subject`
- [ ] Pull Request title contains the target branch if not targeting main: `[0.9.x] KOGITO-XYZ Subject`
- [ ] Pull Request contains link to the JIRA issue
- [ ] Pull Request contains link to any dependent or related Pull Request
- [ ] Pull Request contains description of the issue
- [ ] Pull Request does not include fixes for issues other than the main ticket

<details>
<summary>
How to retest this PR or trigger a specific build:
</summary>

* <b>Pull Request</b>  
  Please add comment: <b>Jenkins retest this</b>
 
* <b>Quarkus LTS checks</b>  
  Please add comment: <b>Jenkins run LTS</b>

* <b>Native checks</b>  
  Please add comment: <b>Jenkins run native</b>

</details>

### CODE REVIEW

**Comment by tiagodolphine** [comment]
jenkins retest this

**Comment by tiagodolphine** [comment]
Jenkins run  kogito-examples tests

**Comment by tiagodolphine** [comment]
jenkins retest this

**Comment by tiagodolphine** [comment]
jenkins retest this

File: process-incubation-api-messaging-quarkus/README.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+### Package and Run in JVM mode
+
+```sh
+mvn clean package
+java -jar target/quarkus-app/quarkus-run.jar
+```
+
+### Submit a request
+
+To make use of this application it is as simple as sending a message to the configured topic ´hello´ with the following content
```
Comment (cristianonicolai) [inline process-incubation-api-messaging-quarkus/README.md:42]: maybe copy similar instructions from https://github.com/kiegroup/kogito-examples/tree/stable/process-kafka-multi-quarkus#use-the-application on how to send the message to the topic and consume it.

**Review by cristianonicolai — COMMENTED** [review]
@tiagodolphine should we also consider adding a native test?

**Review by radtriste — APPROVED** [review]

**Comment by tiagodolphine** [comment]
@evacchi @cristianonicolai wdyt about this example? this could be extended with the new stateful API for processes as well... anyway should we go ahead or just close this?

File: process-incubation-api-messaging-quarkus/pom.xml [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>org.kie.kogito.examples</groupId>
+    <artifactId>kogito-examples</artifactId>
+    <version>2.0.0-SNAPSHOT</version>
+  </parent>
+  <artifactId>process-incubation-api-messaging-quarkus</artifactId>
+  <name>Kogito Example :: Process Incubation API Messaging With Quarkus</name>
+  <properties>
+    <quarkus-plugin.version>2.4.0.Final</quarkus-plugin.version>
```
Comment (cristianonicolai) [inline process-incubation-api-messaging-quarkus/pom.xml:13]: needs update

**Review by cristianonicolai — COMMENTED** [review]
@tiagodolphine just rebase/update with latest Quarkus version, +1 to get it merged and then improve it later on.

**Comment by tiagodolphine** [comment]
closing this, at least for now, it is sitting for a long time, we can open again in case we want to go to showcase this approach of the new API usage.

**Pull request closed without merging**